django/views/debug.py

```diff
@@ -1193,7 +1193,10 @@ def default_urlconf(request):
           </li>
         {% endfor %}
       </ol>
-      <p>The current URL, <code>{{ request_path|escape }}</code>, didn't match any of these.</p>
+      <p>
+        The current URL {% if request_path %}, <code>{{ request_path|escape }}</code>{% endif %}, didn't match
```

Comment:
It would be more correct without the comma in the empty case, and I think a test wouldn't hurt.